olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/VolatileLayerClient.h

```diff
@@ -19,15 +19,17 @@
 
 #pragma once
 
-#include <memory>
-
 #include <olp/core/client/ApiError.h>
 #include <olp/core/client/ApiResponse.h>
 #include <olp/core/client/CancellationToken.h>
 #include <olp/core/client/HRN.h>
 #include <olp/core/client/OlpClientSettings.h>
 #include <olp/dataservice/read/DataRequest.h>
+#include <olp/dataservice/read/PartitionsRequest.h>
 #include <olp/dataservice/read/model/Data.h>
+#include <olp/dataservice/read/model/Partitions.h>
+
+#include <memory>
```

Comment:
As I remember, the STL includes should be on top of the file according to the Google C++ code style